playbooks/cloud-provider-openstack-sanity-test-csi-cinder/run.yaml

```diff
@@ -0,0 +1,18 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-golang
+      go_version: '1.13.4'
```

Comment:
it's good to move this var to `jobs.yaml` so that you can update it in the same place in the future.

Reply:
done